blocks/cards/cards.js

```diff
@@ -24,6 +24,41 @@ export default function decorate(block) {
       const h4title = wDiv.querySelector('h4');
       title.append(h4title);
       wDiv.prepend(title);
+    } else if (block.classList.contains('info-card')) {
+      // Setup an <a> to replace the <div> to make the whole card clickable
+      // Note that this makes it impossible to have the two texts in the
+      // markdown passed link to different places
+      const cardLinkWrapper = document.createElement('a');
+      cardLinkWrapper.classList.add('cards-card-body');
+      const cardDivWrapper = wDiv.querySelector('div.cards-card-body');
+
+      // Add class properties to the elements for styling
+      cardDivWrapper.querySelector('h2').classList.add('header-container', 'card-title');
+      cardDivWrapper.querySelector('p').classList.add('button', 'primary', 'button-container');
+
+      // Get the first link from the markdown and add it to the <> tag as it's href
+      cardLinkWrapper.href = cardDivWrapper.querySelector('h2 > a').getAttribute('href');
+
+      const titleText = cardDivWrapper.querySelector('h2').innerText;
+      const buttonText = cardDivWrapper.querySelector('p').innerText;
+
+      // Pull out the <a> tags from the existing markdown
+      cardDivWrapper.querySelector('h2').removeChild(cardDivWrapper.querySelector('h2 > a'));
+      cardDivWrapper.querySelector('p').removeChild(cardDivWrapper.querySelector('p > a'));
+
+      cardDivWrapper.querySelector('h2').innerText = titleText;
+      cardDivWrapper.querySelector('p').innerText = buttonText;
+
+      // Copy the div HTML contents as the a contents and
+      // replace them in the parent container (the li)
+      cardLinkWrapper.innerHTML = cardDivWrapper.innerHTML;
```

Comment:
Try if it also works with this, would be a bit cleaner:  
```
cardLinkWrapper.append(...cardDivWrapper.children)
```

Reply:
Yup, using the spread operator worked like a charm.  Good tip!  I'll tuck that away in my bag of JS tricks that are slowly getting more modern ;) 

Thanks!